integration-tests/src/test/resources/features/KbvApiHappyPath.feature

```diff
@@ -98,3 +98,52 @@ Feature: 3 out of 4 strategy. User has 2 KBV questions. Tests are run against th
     And a valid JWT is returned in the response
     And a verification score of 2 is returned in the response
     And 10 events are deleted from the audit events SQS queue
+
+  @pre_merge_happy_evidence_requested
+  Scenario: User answers 2 questions correctly with evidence requested
+    Given user has the test-identity 197, verificationScore of 1 and strengthScore of 2 in the form of a signed JWT string
+
+    # Session
+    When user sends a POST request to session end point
+    Then user gets a session-id
+
+    # TXMA event
+    Then TXMA event is added to the SQS queue not containing device information header
+
+    # First question
+    When user sends a GET request to question endpoint
+    Then user gets status code 200
+    And user answers the question correctly
+    Then user gets status code 200
+
+    # Second question
+    When user sends a GET request to question endpoint
+    Then user gets status code 200
+    And user answers the question correctly
+    Then user gets status code 200
+
+    # Third question (will be removed when 2-out-3 strategy is implemented)
```

Comment:
[nit] Can we just add a `TODO` to this so it's really clear it's going to be removed?

Reply:
or maybe we should add as an acceptance criteria on the related ticket

Reply:
That works too - have added comment to https://govukverify.atlassian.net/browse/OJ-2607